### PR TITLE
Feat/home screen

### PR DESCRIPTION
**Pull request by olafch**
# Description (done, pending review)

I've added a new header, subtitle, the two buttons that are present in Docusaurus, and the slider to the website. I've made all the elements responsive. I merged the main branch because I thought I could reuse code from docs (but that was not the case)

## Type of change
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: src/screens/HomeScreen.tsx
Comment (MateuszRostkowski) [inline src/screens/HomeScreen.tsx:1]: This changes should be applied to `LandingScreen` not `HomeScreen` :D

File: src/screens/HomeScreen.web.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  return (
+    <Center flex={1} px={4}>
+      <Text.XxlBold textAlign="center">{t('home_screen.header_title')}</Text.XxlBold>
+      <Text.LgRegular textAlign="center">{t('home_screen.header_subtitle')}</Text.LgRegular>
+
+      <Box
+        alignItems={'center'}
+        flexDirection={'row'}
+        flexWrap={'wrap'}
+        justifyContent={'center'}
```
Comment (MateuszRostkowski) [inline src/screens/HomeScreen.web.tsx:31]: It don't have any impact on code you can change all this props like this:

```suggestion
        justifyContent="center"
```

File: src/screens/index.ts [hunk 2]
```diff
@@ -15,3 +15,4 @@ export * from './SettingsScreen'
 export * from './TestFormScreen'
 export * from './TypographyScreen'
 export * from './UserSessionScreen'
+export * from '../screens/drafts/images'
```
Comment (MateuszRostkowski) [inline src/screens/index.ts:18]:
```suggestion
export * from './drafts/images'
```

File: src/screens/HomeScreen.tsx
Comment (olafch) [reply src/screens/HomeScreen.tsx:1]: Makes sense, thank you Mateusz!

File: src/screens/LandingScreen.tsx [hunk 3]
```diff
@@ -1,38 +1,77 @@
...
+import { Button, Center, Text, Box, ScrollView } from '@baca/design-system'
+import { useCallback, useScreenOptions, useTranslation, useViewportDimensions } from '@baca/hooks'
+import { draftImages } from '@baca/screens'
+import { ImageSourcePropType, Linking, Image, StyleSheet } from 'react-native'
 
 export const LandingScreen = () => {
   const { t } = useTranslation()
+  const { viewportWidth, viewportHeight } = useViewportDimensions()
+
+  useScreenOptions({ title: t('navigation.screen_titles.home') })
```
Comment (MateuszRostkowski) [inline src/screens/LandingScreen.tsx:12]: You have this hook used two times. Just remove this one.

**Review by MateuszRostkowski — CHANGES_REQUESTED** [review]
One little improvment to add, and it is ready to merge! :D 🚀

**Pull request merged**